vendor/github.com/containers/image/docker/archive/dest.go

```diff
@@ -19,15 +19,31 @@ func newImageDestination(ctx *types.SystemContext, ref archiveReference) (types.
 	if ref.destinationRef == nil {
```

Comment:
@rhatdan @umohnani8 okay, I think we should mention that in the PR comments lest it gets merged by mistake.